example/android/app/build.gradle

```diff
@@ -34,7 +34,7 @@ android {
     defaultConfig {
         // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
         applicationId "com.dooboolab.fluttersoundexample"
-        minSdkVersion 16
+        minSdkVersion 24
```

Comment:
Why is this necessary to change the minSdkVersion in `example`?

Reply:
This is because flutter_ffmpeg works on minSdkVersion 24 (or later).
If the user does not need OPUS he can continue to use SDK 16.
But if it is necessary to have SDK 24 for ffmpeg.

The example needs OPUS to test how it works.

Note: Actually, ffmpeg is not used on Android. But we probably will use it later, for example to use `lame` for encoding MP3

Reply:
Could you also provide this detail in the readme for other users of flutter_sound?

Reply:
flutter_ffmpeg LTS Release - minAndroidSdk 16 (instead of 24)

I was able to build the Example App with MinAndroidSdk 16, using the LTS Release of flutter_ffmpeg instead of the Main Release